scripts/download_file.py

```diff
@@ -0,0 +1,156 @@
+
+import argparse
+import urllib.request
+import urllib.error
+import os
+import hashlib
+import sys
+
+
+parser = argparse.ArgumentParser(description="Download data file for EFI KBase app")
+parser.add_argument("--remote-dir", required=True, type=str, help="URL of the directory in which the file resides")
+parser.add_argument("--remote-file", required=True, type=str, help="Name of the file on the remote to download")
+parser.add_argument("--local-dir", required=True, type=str, help="Path to local directory where file will be unpacked")
+parser.add_argument("--local-file", required=True, type=str, help="Path to file on local system where download file is placed")
+args = parser.parse_args()
+
+
+if not os.path.isdir(args.local_dir):
+    os.mkdir(args.local_dir)
+
+
+
+
+
+
+def get_remote_file_contents(remote_file):
+    """
+    Downloads the file as a string and return to user; if HTTP error then returns False
```

Comment:
add parameter and return sections to every docstring and consider type-annotating your functions